### ISSUE TITLE
Meeting 04-12-2023

### ISSUE THREAD

**Issue by maiklarooij** [BluntKatana/fact-checker#1]
# Notes Maik & Floris
- Found a working **LLM** -> Zephyr 7B (outperforms LLaMa 70B model)
- Experiments with **Entity Linking**: words that are closer to the mention score more points
- **Classifiying questions** as 'yes/no' or 'entity'
    - First approach: look at 'where', 'what', 'who' etc. at the beginning of the question
    - Now working on: classifier trained on on 150K+ questions and type datasets
- **Answer extraction**
    - 'yes/no' question: pre-trained classifier from [boolq](https://huggingface.co/datasets/boolq) dataset (question, context, answer), seems to work pretty well
    - 'entity' question: for now, return the first found entity in the answer that is not in the question 
- Start creating a test set
